iotdb-core/datanode/src/assembly/resources/sbin/remove-datanode.bat

```diff
@@ -22,18 +22,14 @@
 IF "%~1"=="--help" (
     echo The script will remove one or more DataNodes.
     echo Before removing DataNodes, ensure that the cluster has at least the number of data/schema replicas DataNodes.
-    echo Usage:
-    echo Remove one or more DataNodes with datanode_id
-    echo ./sbin/remove-datanode.bat [datanode_id ...]
+    echo Usage: ./sbin/remove-datanode.sh [DataNode_ID ...]
+    echo Remove one or more DataNodes by specifying their IDs.
+    echo Example:
+    echo   ./sbin/remove-datanode.sh 1         # Remove DataNode with ID 1
```

Comment:
Note that this datanode is deleted by default if it is not specified.

Reply:
resolved